src/utils/DynamicContent/DynamicCampaignText.ts

```diff
@@ -32,6 +34,7 @@ export default class DynamicCampaignText implements DynamicContent {
 		this._campaignParameters = campaignParameters;
 		this._impressionCount = impressionCount;
 		this._cache = new Map<string, string>();
+		this.currentTime = this.currentTime.bind( this );
```

Comment:
Why is this needed?

Reply:
Because we're using a class, and the method uses `this` internally, we need to bind the method to the current scope when it's constructed. Otherwise we'll get 'this is not defined' errors.